Firestore/Example/Tests/Local/FSTLRUGarbageCollectorTests.h

```diff
@@ -17,12 +17,13 @@
 #import <XCTest/XCTest.h>
 
 @protocol FSTPersistence;
+struct FSTLruGcParams;
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface FSTLRUGarbageCollectorTests : XCTestCase
 
-- (id<FSTPersistence>)newPersistence;
+- (id<FSTPersistence>)newPersistenceWithLruGcParams:(FSTLruGcParams)lruGcParams;
```

Comment:
This code includes both LRU and Lru spellings of the acronym for least recently used. It also includes GarbageCollector, Gc, and GC. It's a mess.

I realize you're trying to avoid FSTLRUGC as the start of anything, and I can appreciate that :-).

It seems doubtful to me that we're going to end up with more than one LRU-anything in Firestore or at least in the local area. For LRU-GC-related types, could just use LRU? (e.g. could "LRU GC parameters" just be "LRU parameters"? Or "LRU options"?)

When used in a label you can even shorten it to just "options", like so:

```objc
- (id<FSTPersistence>)newPersistenceWithOptions:(FSTLRUOptions)options;
```

Reply:
Switch to `LruOptions` and `LruResults`, and they are both in C++ namespaces.